internal/client.go

```diff
@@ -56,8 +56,8 @@ func (c *Client) init() error {
 	var p parser.Markdown
 	p.Init()
 	c.parser = p
-
-	f := provider.File{Path: c.Path, Parser: p}
+	h := provider.FileHelpersC{}
```

Comment:
If you make this struct private, you don't have to initialize it here. You could just declare it at the struct and we're good.

Reply:
Thanks. Fixed.